packages/svelte/tests/compiler-errors/samples/css-global-block-combinator-2/main.svelte

```diff
@@ -0,0 +1,11 @@
+<style>
+	/* ok */
+	.x > :global.x > p {
+	}
```

Comment:
this seems a bit weird — isn't that equivalent to this, which is sort of meaningless?

```css
.x > :global {
  &.x > p {...}
}
```

Given that the `:global` gets removed, the `&` refers to the combinator which makes no sense.
I note that in the [playground](https://svelte-5-preview-git-css-global-tweaks-v2-svelte.vercel.app/#H4sIAAAAAAAAE4WOQQ7CIBREr0L-2tg9IonnEBcIv4bkFwjQpg3h7oKuXLmbeZnJTIXZEWbg9wpeLwgcbjHCCcoRh8kbUsHuc1iTGURYtzFDOuergl2BFFMnUnnlRS4H4ZDlvDPJ-IvCU9NHR1YHV8UEComzhPYyQBvF3zz7VxDTd6jfWoJ1s0MLvKQV26O9AVzjuWHRAAAA), these turn into the same thing, which tells me something is wrong:

```css
.x > :global.x > p {...}
.x > :global .x > p {...}
```

I think the rule should be that `:global` can only follow the descendant combinator (unless it's the first thing i.e. there is no combinator)

Reply:
(in fact it looks like we already had that rule?)

Reply:
We had but I thought it's too restrictive, but you're right that it introduces inconsistencies, will revert